packages/info/logger/index.js

```diff
@@ -39,23 +57,47 @@ class Logger {
     const prefix = colorize(`${name}:warning`, 'yellow');
     const message = getCleanMessage(warning);
     if (level >= logLevels.warn) {
-      console.warn(`${prefix} ${format(message, ...args)}`);
+      const formatted = `${prefix} ${format(message, ...args)}`;
+      console.warn(formatted);
+      if (this.logLocation) {
+        appendFile(
+          join(this.logLocation, 'hops-log.txt'),
+          `${new Date().toISOString()} ${formatted}\n`,
+          noop
+        );
+      }
     }
   }
 
   info(message, ...args) {
     const { level, name } = this;
     const prefix = colorize(`${name}:info`, 'gray');
     if (level >= logLevels.info) {
-      console.log(`${prefix} ${format(message, ...args)}`);
+      const formatted = `${prefix} ${format(message, ...args)}`;
+      console.log(formatted);
+      if (this.logLocation) {
```

Comment:
`this.maybeWriteLog(formatted)`

und dann:

```js
maybeWriteLog(formatted) {
  if (!this.logLocation) {
    return 
  }

  appendFile(
    join(this.logLocation, 'hops-log.txt'),
    `${new Date().toISOString()} ${formatted}\n`,
    noop
  );
} 

```

Reply:
Nö :joy: